pkg/updater/trainingJobUpdater.go

```diff
@@ -2,15 +2,16 @@ package updater
 
 import (
 	"fmt"
+	"reflect"
+	"time"
+
 	log "github.com/golang/glog"
```

Comment:
These import paths are not sorted lexically. This violates the Go code style convention.  You might need to configure your favorite editor to call either `gofmt` or `goimports` whenever you save the .go files.

Are you using Emacs? If so, a very convenient configuration is https://github.com/helinwang/go-emacs.